src/contracts/ownership/InitializedOwnable.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.7.5;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title Ownable contract with custom intial owner
+ * @author Gnosis Developers
+ * @dev A contract extending Openzeppelin's Ownable contract that allows to
+ * specify the initial owner in the contructor instead of using the message
+ * sender.
+ */
+abstract contract InitializedOwnable is Ownable {
```

Comment:
nanonit (feel free to ignore): I think the name could emphasize more clearly how it differs from the standard Ownable that most developers know. Technically OZ's Ownable is also initialized. Maybe Naming it CustomInitiallyOwnable or OwnableByExternal, OwnableByThirdPary would be slightly more descriptive.

Reply:
Or, similarly and shorter, ExternallyOwnable.

Reply:
My issue with `ExternallyOwnable` or `OwnableByExternal` is that this is already in general true for `Ownable` (as the ownership can be transferred). Personally, I actually like `InitializedOwnable` as its an "`Ownable` initialized with an `owner`".